apps/prairielearn/src/lib/editorUtil.ts

```diff
@@ -53,15 +69,13 @@ export function getDetailsForFile(filePath) {
   }
 }
 
-/**
- * @param {any} courseId
- * @param {string} filePath
- * @returns {Promise<{ errors: string | null, warnings: string | null }>}
- */
-export async function getErrorsAndWarningsForFilePath(courseId, filePath) {
+export async function getErrorsAndWarningsForFilePath(
+  courseId: any,
+  filePath: string,
+): Promise<{ errors: string | null; warnings: string | null }> {
   const details = getDetailsForFile(filePath);
-  let queryName = null;
-  let queryParams = { course_id: courseId };
+  let queryName: string | null = null;
+  const queryParams: QueryParams = { course_id: courseId };
```

Comment:
Nit, this can probably just be `Record<string, any>`. You could also use `QueryParams` from `@prairielearn/postgres`. We don't benefit from any more type-safety here since this all just gets dumped into `queryZeroOrOneRowAsync`.